apps/designer/app/shared/uploads/array-buffer-helpers.ts

```diff
@@ -0,0 +1,25 @@
+export const mergeUint8Arrays = (a: Uint8Array, b: Uint8Array) => {
+  const c = new Uint8Array(a.length + b.length);
+  c.set(a);
+  c.set(b, a.length);
+
+  return c;
+};
+
+export const getFilenameAndExtension = (filename: string) => {
```

Comment:
for this there are node built-in functions

Reply:
there is for extension, changed it